danielMunteanu/src/main/java/danielCollectionTask/CompanyOrder.java

```diff
@@ -0,0 +1,55 @@
+package danielCollectionTask;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static java.time.LocalDateTime.now;
+
+public class CompanyOrder implements Order {
+    final long id;
+    private Company company;
+    private List<Item> items;
+    private int counter;
+    final LocalDateTime date;
+
+    public CompanyOrder() {
+        this.date = now();
+        this.id = ++counter;
+    }
+
+    @Override
+    public String toString() {
+        return "danielCollectionTask.Order{" +
+                "id=" + id +
+                ", date='" + date.toString() + '\'' +
+                ", user='" + company + '\'' +
+                ", items=" + items +
+                '}';
+    }
+
+    @Override
+    public double sum() {
```

Comment:
duplicated code. Please, move all the common code into a single module (Class)

Reply:
removed